### DISCUSSION TITLE
use Giscus. Get 404

### DISCUSSION THREAD

**Discussion by CooperJiang** [vuepress-theme-hope/vuepress-theme-hope#2085] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/49814706/180928274-349033cc-0ead-4869-96c5-169ddaa875b0.png)
![image](https://user-images.githubusercontent.com/49814706/180928344-9b25fb89-c593-4549-a5d9-5b9b8e386ce5.png)

**Comment by Mister-Hope** [comment, marked as answer]
Could you PLEASE read giscus FAQ?

**Comment by lingwh1995** [comment]
测试
